web/styles/compose.css

```diff
@@ -1202,6 +1203,28 @@ textarea.new_message_textarea {
     }
 }
 
+#compose_recipient_box #recipient_box_clear_topic_button,
+#compose-direct-recipient #add_dm_recipient_button {
```

Comment:
I think picking a class name for this concept, that both code paths can share, would be good. Maybe `.compose-new-recipient-button` would be a good name?

Also `add_dm_recipient_button` uses legacy `_`s and isn't a well-scoped name, let's do `compose-new-direct-recipient-button`.